combat-trainer.lic

```diff
@@ -71,6 +71,7 @@ class SetupProcess
     was_retreating = game_state.retreating?
     game_state.update_room_npcs
     if game_state.dancing?
+      game_state.sheath_whirlwind_offhand
```

Comment:
Will the whirlwind offhand be wielded again after dancing? As in, is this fixing one issue but introducing a new problem when resuming whirlwind?